client/hosting-overview/components/quick-actions-card.tsx

```diff
@@ -72,6 +83,11 @@ const QuickActionsCard: FC = () => {
 					href={ statsUrl }
 					text={ translate( 'See Jetpack Stats' ) }
 				/>
+				<Action
+					icon={ <SidebarCustomIcon icon="dashicons-wordpress-alt hosting-overview__dashicon" /> }
+					href={ site ? site.options?.admin_url || `${ site.URL }/wp-admin` : '' }
+					text={ translate( 'WP Admin' ) }
```

Comment:
Hmm, would it make more sense to have this as the first item? Somehow looks weird to have WP Admin link as the last one...

Reply:
Sure 🙂